packages/framework/src/Controller/Admin/DefaultController.php

```diff
@@ -56,6 +60,8 @@ public function __construct(
         protected readonly BreadcrumbOverrider $breadcrumbOverrider,
         protected readonly DateTimeFormatterExtension $dateTimeFormatterExtension,
         protected readonly TransferIssueFacade $transferIssueFacade,
+        protected readonly LegalConditionsFacade $legalConditionsFacade,
+        protected readonly Domain $domain,
```

Comment:
This is considered a BC break (see https://symfony.com/doc/current/contributing/code/bc.html#changing-classes). 
Can you please document the change in the upgrade notes and update the PR description (BC Break: yes)?

Reply:
Both done.